src/test/java/com/devonfw/sample/archunit/PackageRuleTest.java

```diff
@@ -0,0 +1,76 @@
+package com.devonfw.sample.archunit;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+
+import com.tngtech.archunit.core.domain.JavaClass;
+import com.tngtech.archunit.core.importer.ImportOption;
+import com.tngtech.archunit.junit.AnalyzeClasses;
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.lang.ArchCondition;
+import com.tngtech.archunit.lang.ArchRule;
+import com.tngtech.archunit.lang.ConditionEvents;
+
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import com.tngtech.archunit.lang.SimpleConditionEvent;
+
+/**
+ * JUnit test that validates the Packages of this application.
+ */
+
+
+@AnalyzeClasses(packages = "com.devonfw.sample.archunit", importOptions = ImportOption.DoNotIncludeTests.class)
+public class PackageRuleTest {
+
+/*Components */
+public static final String COMPONENT_GENERAL = "general";
+
+public static final String COMPONENT_TASK = "task";
```

Comment:
I am not sure if it is not a good idea to hardcode the components into the regex.
We only need the actual component names in the component dependency rules but for the package rule there is no point in restricting the components and listing all of them here as constants... As a result this rule will not need to be customized by the projects.
If we need the component name we can get it from a capturing group of the regex.
When implementing #3 we will see if we want to build it on top of this and your regex or if we use archunit features directly.